### PR TITLE
Fix notebook toolbar when ADS screen width is decreased

### PR DESCRIPTION
**Pull request by lucyzhang929**
This PR fixes https://github.com/microsoft/azuredatastudio/issues/15895

Screen Shot 2021-07-02 at 1 37 05 PM

### CODE REVIEW

File: src/sql/workbench/contrib/notebook/browser/notebook.css [hunk 1]
```diff
@@ -38,7 +38,7 @@
 }
 
 .notebookEditor .in-preview .actions-container .action-item {
-	height: 100%;
+	height: 21px;
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/browser/notebook.css:41]: Was 21px the height we had before? And if not where was it getting its height from before?

Comment (lucyzhang929) [reply src/sql/workbench/contrib/notebook/browser/notebook.css:41]: I believe it was coming from here before https://github.com/microsoft/azuredatastudio/blob/426d230871824e13a9f05f2042d65315e56b9a81/src/sql/workbench/contrib/notebook/browser/notebook.css#L55

Comment (Charles-Gagnon) [reply src/sql/workbench/contrib/notebook/browser/notebook.css:41]: Could you just remove this rule then? Seems like the other one should take effect at that point.

Comment (lucyzhang929) [reply src/sql/workbench/contrib/notebook/browser/notebook.css:41]: Actually that rule is used for the non-preview toolbar icons, so removing that would affect those icons.

Comment (Charles-Gagnon) [reply src/sql/workbench/contrib/notebook/browser/notebook.css:41]: I was suggesting removing this rule - not the one on line 55. 

I played around with it a bit and it seems like if you :

1. Remove the rule on [line 40](https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/notebook/browser/notebook.css#L40)
2. Remove the height: 100% on [line 65 ](https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/notebook/browser/notebook.css#L65)

Which appears to also fix it, but also has the advantage of using the same rules for both preview and non preview icons.

Comment (lucyzhang929) [reply src/sql/workbench/contrib/notebook/browser/notebook.css:41]: Ah yes that fixes the issue. Thank you for the help Charles!

**Review by Charles-Gagnon — APPROVED** [review]

**Review by chlafreniere — APPROVED** [review]

**Pull request merged**